scripts/cpg_cromwell/remove_analysis_outs.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
This script is getting close to the border of being too permissive to delete and move files to release because it relies on user input here. As it allows moving to and deleting from release and this repository and datasets this belongs to is within your control, I'm happy to leave it, but see the comment below.